README.md

```diff
@@ -113,6 +113,10 @@ Note that you need to add `data-sophi-feature=<widget_name>` to the wrapper div
 </div>
 ```
 
+#### Caveats
+
+While the above query integration works just fine, it has been observed on [WordPress VIP](https://wpvip.com/) infrastructure that `WP_Query` may return latest posts instead of the posts curated by Sophi. A workaround for this is to use [`get_posts()`](https://developer.wordpress.org/reference/functions/get_posts/) instead.
```

Comment:
It may be worth also calling out here that should an integration need to switch to using `get_posts()` that we'd recommend a strong comment in that codebase to call out WHY so that some future engineer doesn't come along and swap it out for `WP_Query` and totally break things.